### DISCUSSION TITLE
Atari PCS (8-bit Family) — 8bitworkshop documentation

### DISCUSSION THREAD

**Discussion by giscus** [sehugg/8bitworkshop#134] [General]
# Atari PCS (8-bit Family) — 8bitworkshop documentation

Atari PCS Specifications: Lifespan, 1984-1992,, Media, Floppy disk, cassette tape, ROM cartridge,, CPU	8-bit, MOS Technology 6502 @ 1.79 (NTSC) / 1.77 (PAL) MHz,, Memory, 16 - 128 KB dynamic RAM,, ...

https://8bitworkshop.com/docs/platforms/atari8/index.html

**Comment by Sasatari** [comment]
Lifespan is wrong. The Atari 400 and 800 were introduced 1979.
CPU is a 6502C, called "Sally".
